.circleci/config.yml

```diff
@@ -138,21 +159,37 @@ jobs:
 
             [ -n "${BACKWARD_COMPATIBILITY}" ] && CHECK_BACKWARD="false" || CHECK_BACKWARD="true"
             ./Tests/scripts/validate.sh
+
+            if [ $? -ne 0 ] && [ -n "${DEMISTO_SDK_NIGHTLY}" ] ;
```

Comment:
need to make sure that if the flake8 fails, we also fail the step
in this state - I'm not sure this is true.

Reply:
Done